resources/models/reservation.py

```diff
@@ -118,6 +119,7 @@ class Reservation(ModifiableModel):
                                                               null=True)
     participants = models.TextField(verbose_name=_('Participants'), blank=True)
     host_name = models.CharField(verbose_name=_('Host name'), max_length=100, blank=True)
+    reservation_extra_questions = models.TextField(verbose_name=_('Reservation extra questions'), null=True, blank=True)
```

Comment:
Is there a specific need for this to be `null=True`? Other TextFields here are only `blank=True`.

Reply:
You're right, `null=True` on `TextField` is unnecessary and may cause confusion. I'll change it!